apps/web/app/(navigation)/(multiCard)/layout.tsx

```diff
@@ -0,0 +1,81 @@
+"use client";
+import { CardWrapperStyled, CardStyled } from "../HomePage";
+import { LayoutProps } from "app/layout";
+import { device } from "lib/media";
+import { CheckboxWrapper } from "app/CheckboxWrapper";
+import styled from "styled-components";
+
+const RightCardStyled = styled(CardStyled)`
+  display: none;
+  @media ${device.desktop} {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-self: flex-start;
+    flex-basis: 15%;
+    min-width: 300px;
+    min-height: 250px;
+    margin-left: 1%;
```

Comment:
I think it is better to change it to 2% (the padding value of the parent) and keep the same spaces around the cards. Or rather, use ```gap: 2%;``` in ```CardWrapperStyled``` instead of the margin here ;D
```suggestion
    margin-left: 2%;
```
![image](https://user-images.githubusercontent.com/38572172/229630169-b383964a-6ccd-4454-9862-9a1ffb5ae0cb.png)

Reply:
I was tought that % and vh, vv used in RWD layouts is a big no no.
So i recommend to set margins to constant values.

Reply:
@Noshi96 great advice,thanks! Is that a rule only for margins and paddings?

Reply:
@Noshi96 it's done ☺️